airflow/dag_processing/manager.py

```diff
@@ -725,7 +726,7 @@ def _refresh_dag_dir(self):
 
             try:
                 self.log.debug("Removing old import errors")
-                self.clear_nonexistent_import_errors()
+                self.clear_nonexistent_import_errors(file_paths=self._file_paths)
```

Comment:
It is a static method now

```suggestion
                DagFileProcessorManager.clear_nonexistent_import_errors(file_paths=self._file_paths)
```